lib/transforms/minifyCss.js

```diff
@@ -0,0 +1,33 @@
+const cssnano = require('cssnano');
+
+const defaultQueryObj = {type: 'Css', isLoaded: true};
+
+module.exports = queryObj => {
+    return async assetGraph => {
+        if (queryObj) {
+            queryObj = assetGraph.query.and(queryObj, defaultQueryObj);
+        } else {
+            queryObj = defaultQueryObj;
+        }
+        for (const cssAsset of assetGraph.findAssets(queryObj)) {
+            const result = await cssnano.process(cssAsset.parseTree, undefined, {
```

Comment:
@papandreou: Should `cssnano.process` be wrapped in a try/catch block with errors being passed to `assetgraph.warn`?

Reply:
@davej, hmm, I guess that can't hurt? Haven't ever seen it return a rejected promise, though.

Reply:
https://github.com/assetgraph/assetgraph/commit/ea7c1e4327a908bfe77f365f19415fa4020d3eb1

Reply:
@papandreou Hmm, does it not reject if the input css is malformed or non-css content.

Reply:
Ah, you're right, although that would come from the execution of `cssAsset.parseTree` if the asset hasn't already been parsed. I've added a test of that scenario now: https://github.com/assetgraph/assetgraph/commit/12afea957f364e8b02997b704c302f5db0ae2b05